### ISSUE TITLE
Downsides and important usage notices of apps like encryption are unvisible before activation

### ISSUE THREAD

**Issue by Raydiation** [owncloud/core#17779]
I've seen countless users that activate the encryption app without knowing what it actually does and what has to be considered e.g. that it's possible for people to decrypt the files if they've got access to your server, that it only encrypts new files and that the file size goes up by ~35%

Since encryption 2.0 launched the app was renamed to **Default encryption module** which does not communicate what the app actually does. Furthermore in 8.1 the description is hidden by default and has to be expanded by clicking on it, which basically encourages users to not read the included description. If you actually click on it, it's expands into an unformatted wall of text which is hard to read. It almost seems like we want to trick users into enabling it ;D

Apart from that people don't read FAQs, documentation or blog posts for the same reason they don't read Terms of Services or EULAs. I think we should use https://tosdr.org/ as a good example (as suggested a while ago by @jancborchardt ) on how we should structure those things:

* Use short sentences
* Use bullet points since they increase readability by at least 100% (can't find the source anymore, heard it during my UX lectures on university) 
* Use colors and icons to highlight important parts

Further information http://thecontentauthority.com/blog/content-readability-the-success-factor-you-cant-ignore

PS: these things would also apply to other apps like the News app which requires you to set up a cronjob

TL;DR: **We need to make sure that users know about issues and problems before activating an app**

Any ideas?

@jancborchardt @Henni @schiesbn @MorrisJobke @PVince81 @LukasReschke @jospoortvliet

**Comment by Raydiation** [comment]
Another thing: the way it is now is maximum punishing for people who have enabled it without reading the docs because:
* No docs on how to manually decrypt files
* Files are not decrypted if the app is disabled

**Comment by oparoz** [comment]
I think it's even worse with encryption 2.0 as you can't get the original files any more.

**Comment by karlitschek** [comment]
@schiesbn @carlaschroder

**Comment by jospoortvliet** [comment]
I would suggest to show the first 3 lines of text from a description. And yes, descriptions should support markdown or something...

The description of the Encryption app isn't bad, but the name "default encryption module" is not exactly ideal. "Default Server Side Encryption module" is long but I think that the 'server side' element is very important. At least it makes it clear that this isn't needed to enable SSL/https!

**Comment by schiesbn** [comment]
>The description of the Encryption app isn't bad, but the name "default encryption module" is not exactly ideal. "Default Server Side Encryption module" is long but I think that the 'server side' element is very important. At least it makes it clear that this isn't needed to enable SSL/https!

Just as a note: If people just enable the "default encryption module" but not server-side encryption in the admin settings nothing will happen. So at least the impact on enabling the app by accident can be ignored in my opinion. But I will look at the app description for improvements

**Comment by schiesbn** [comment]
> Furthermore in 8.1 the description is hidden by default and has to be expanded by clicking on it, which basically encourages users to not read the included description.

I agree. This is also something I don't really like about the app menu. It hides really important information and makes it quite hard to discover it at all. But I also see the visual argument by @jancborchardt, but maybe we can still find ways to improve it?

**Comment by jancborchardt** [comment]
@schiesbn for encryption, we could put the »Enable« button inside the part which gets expanded with the description. Or disable unless the description is expanded, showing a tipsy tooltip about it when hovered or clicked. That would force people to read the stuff.

**Comment by jospoortvliet** [comment]
@jancborchardt that would be a good idea, perhaps for all apps...

**Comment by jancborchardt** [comment]
@jospoortvliet no, I don’t want to force people to open the description, that would make app installation a chore. See the Play Store, iOS app store, Ubuntu Software Center, Firefox Marketplace etc – they all allow quick installation without entering the detail view. Most of the time the description is not needed anyway.

**Comment by jospoortvliet** [comment]
allright, good with me - as long as we can solve this for this app, I'm good. The proposed solution works for me.

**Comment by carlaschroder** [comment]
Not really seeing a doc issue here-- when you have some specifics please open an issue in the doc repo.

**Comment by Raydiation** [comment]
PR is already open btw, reviews please

**Comment by jospoortvliet** [comment]
Link to PR? Isn't it merged already?

**Comment by LukasReschke** [comment]
@josh4trunks https://github.com/owncloud/core/pull/17810
